host/platform/Windows/capture/NVFBC/src/nvfbc.c

```diff
@@ -350,6 +350,8 @@ static CaptureResult nvfbc_getFrame(FrameBuffer * frame)
 
 static int pointerThread(void * unused)
 {
+  bool hasPosition = true;
```

Comment:
Always true? This should be a member variable, and in `static void on_mouseMove(int x, int y)`

Reply:
Good catch, I meant false.

Reply:
Even still, this should be set in `on_mouseMove`

Reply:
Done.